### DISCUSSION TITLE
怎么缩小编码区域与候选区域的间距？

### DISCUSSION THREAD

**Discussion by kwindva** [rime/home#1724] [Q&A] [answered]
取消了预编辑区（preedit），然后把style: layout: spacing设置成0，结果如下图，还是觉得上面的拼音和下面的文字，距离太远，还能怎么缩小它们之间的距离呢？spacing改成负数好像也是按正数处理的。。
![图片](https://github.com/user-attachments/assets/d474ac31-2988-46b2-8ffa-70b0dd8e1684)

**Comment by lingyust** [comment]
我也有这个疑问，设置spacing似乎没有任何效果，编码区和候选区如果紧凑点的话，会更好看一些
![1](https://github.com/user-attachments/assets/c3cd9573-04c8-4bdf-b590-de9c68c11a71)

**Comment by lotem** [comment]
不太瞭解.
我懷疑文字本身要佔那麼大高度.
大部分字體都不是頂格寫. 拉丁字母也不是貼着地寫的, 基線以下也是文字行高的一部分.
spacing 可以增大間距, 但不能讓兩行文字的渲染區域重疊.

**Reply by lingyust** [reply, marked as answer]
已经找到原因了，是高亮区和内部文字之间的距离设置的过大。
修改hilite_padding_y: 8，这行代码是被注释掉的，可以将8修改的小一些，我设置成了2，看起来就好多了

**Reply by kwindva** [reply]
> 已经找到原因了，是高亮区和内部文字之间的距离设置的过大。 修改hilite_padding_y: 8，这行代码是被注释掉的，可以将8修改的小一些，我设置成了2，看起来就好多了

问题解决，非常感谢。
